packages/SwingSet/docs/dynamic-vats.md

```diff
@@ -158,13 +158,13 @@ E(adminNode).done()
 
 When the vat halted due to a metering fault, `error` will be a `RangeError` with a message of `Compute meter exceeded`, `Allocate meter exceeded`, or `Stack meter exceeded`.
 
-## Upgrade
+## Restart
 
-Dynamic vats can be upgraded to use a new source code bundle. Most vat state is discarded, however "durable" collections are retained for use by the replacement version. For full details, see [vat-upgrade.md](./vat-upgrade.md).
+Dynamic vats can be restarted to use a new source code bundle. Most vat state is discarded, however "durable" collections are retained for use by the replacement version. For full details, see [vat-upgrade.md](./vat-upgrade.md).
```

Comment:
"dynamic vats can be restarted, with or without a new source code bundle"

Reply:
"restart" is also used in the sense of: restart from snapshot, preserving the heap

"upgrade" doesn't preserve the heap. renaming it isn't a bad idea, but "reboot" would be easier for my tiny brain, so as not to collide with "restart"